build.sbt

```diff
@@ -17,4 +18,32 @@ lazy val `scala-tutorial` = (project in file("."))
         "com.github.alexarchambault" %% "scalacheck-shapeless_1.12" % "0.3.1",
         compilerPlugin("org.spire-math" %% "kind-projector" % "0.7.1")
       )
-    )
\ No newline at end of file
+    )
+
+// Distribution
+
+lazy val gpgFolder = sys.env.getOrElse("PGP_FOLDER", ".")
+
+lazy val publishSettings = Seq(
+  organizationName := "Scala Exercises",
+  organizationHomepage := Some(new URL("http://scala-exercises.org")),
+  startYear := Some(2016),
+  description := "Scala Exercises: The path to enlightenment",
```

Comment:
We should make all these frequently copied and pasted settings a common setting shared among exercise projects